protobuf-support/src/lexer/lexer_impl.rs

```diff
@@ -67,6 +67,15 @@ impl From<ProtobufFloatParseError> for LexerError {
     }
 }
 
+/// The raw bytes for a single char or escape sequence in a string literal
+///
+/// The raw bytes are available via an `into_iter` implementation.
+pub struct DecodedBytes {
```

Comment:
This seems to be not used outside of the crate, so it should not be public.

Reply:
It's the return type of a public method, so it needs to be pub. We could modify that signature to return `impl Iterator`, if that is preferable?